applications/luci-app-sshtunnel/htdocs/luci-static/resources/view/sshtunnel/ssh_hosts.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+'require form';
+'require fs';
+'require ui';
+'require view';
+
+
+return view.extend({
+	load: function () {
+		return Promise.all([
+			fs.lines('/root/.ssh/known_hosts'),
+		]);
+	},
+
+	render: function (data) {
+		var knownHosts = data[0];
+
+		var m, s, o;
+
+		m = new form.Map('sshtunnel', _('SSH Tunnels'),
+			_('This configures <a href="https://openwrt.org/docs/guide-user/services/ssh/sshtunnel">SSH Tunnels</a>.')
```

Comment:
Please separate this link out to a `.format()`
This is how I did it for another comment:
`			_('Discard upstream responses containing <a href="%s">RFC1918</a> addresses.').format('https://datatracker.ietf.org/doc/html/rfc1918'));
`